modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -923,6 +923,20 @@ The product's default behavior of using a cost-based IDP search algorithm when c
 For more information, see link:https://neo4j.com/docs/cypher-manual/current/query-tuning/query-options/#cypher-planner[Cypher manual -> Cypher planner].
 ====
 
+.Using the unicode \u0085 in an unescaped identifier
+====
+Query::
++
+[source,cypher]
+----
+RETURN 1 as my\u0085identifier
+----
+Description of the returned code::
+The unicode `\\u0061` is deprecated in unescaped identifiers.
```

Comment:
This sounds like only that particular Unicode `\\u0061` is deprecated. I guess all Unicodes must be escaped when used as identifiers.

Reply:
No, you can still use other unicodes. `RETURN 1 as my\u0061identifier` is parsed as `RETURN 1 as myaidentifier` at is valid.